tests/integration/test_vad_gradient_step_and_eval.py

```diff
@@ -0,0 +1,190 @@
+# ! /usr/bin/python
+# -*- coding: utf-8 -*-
```

Comment:
These tests are fine, but add a metric test for the confusion matrix if possible.

Reply:
Could you please show me where we test other metrics?

Reply:
We dont test it yet, but confusion matrix might need checks to ensure that input is binary (if we chose not to support multiclass confusion matrix) or that subsequent code doesnt break if we do pass a multiclass confusion matrix to compute the F1/Prec/Recall scores.